packages/xo-web/src/xo-app/new-vm/index.js

```diff
@@ -1781,7 +1781,7 @@ export default class NewVm extends BaseComponent {
             <SectionContent>
               <Item label={_('secureBoot')}>
                 <Toggle onChange={this._toggleState('secureBoot')} value={secureBoot} />
-                {secureBoot && !pool.secureBootSetup && (
+                {secureBoot && !pool?.secureBootSetup && (
```

Comment:
`!undefined` return `true`.
Do something like that instead: 
`secureBoot && pool !== undefined && !pool.secureBootSetup` or `secureBoot && pool?.secureBootSetup === false`